### PR TITLE
feat: update readme

### PR DESCRIPTION
**Pull request by yn-mrse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
-# ad_status_lamp_manager
+# autonomous_driving_status_lamp_manager
```
Comment (MasaakiWatanabeTier4) [inline README.md]:
```suggestion
# Status lamp manager for autonomous driving
```

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+
+## Overview
+By switching the lighting pattern of the lamp, this node informs the transport manager of the system status.
```
Comment (MasaakiWatanabeTier4) [inline README.md:4]: 要検討

File: README.md [hunk 3]
```diff
@@ -1 +1,50 @@
...
+
+The lighting pattern is defined as follows.
```
Comment (MasaakiWatanabeTier4) [inline README.md]:
```suggestion
Lighting patterns are defined as follows;
```

File: README.md [hunk 4]
```diff
@@ -1 +1,50 @@
...
+<table>
+  <thead>
+    <tr>
+      <th scope="col">system status</th>
+	    <th scope="col">lighting pattern</th>
+	  </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>During system startup</td>
```
Comment (MasaakiWatanabeTier4) [inline README.md]:
```suggestion
      <td>System starting up</td>
```

File: README.md [hunk 5]
```diff
@@ -1 +1,50 @@
...
+      <td>Blinks every second</td>
```
Comment (MasaakiWatanabeTier4) [inline README.md]:
```suggestion
      <td>Blink once every second</td>
```

File: README.md [hunk 6]
```diff
@@ -1 +1,50 @@
...
+    </tr>
+    <tr>
+      <td>Preparing for autonomous driving</td>
+      <td>Flashes twice quickly on a regular basis</td>
```
Comment (MasaakiWatanabeTier4) [inline README.md:21]:
```suggestion
      <td>Flash twice a second</td>
```

File: README.md [hunk 7]
```diff
@@ -1 +1,50 @@
...
+    </tr>
+    <tr>
+      <td>Autonomous driving is available</td>
+      <td>Keeps lighting</td>
```
Comment (MasaakiWatanabeTier4) [inline README.md:25]:
```suggestion
      <td>Light up</td>
```

File: README.md [hunk 8]
```diff
@@ -1 +1,50 @@
...
+    </tr>
+    <tr>
+      <td>During system shutdown</td>
```
Comment (MasaakiWatanabeTier4) [inline README.md:28]:
```suggestion
      <td>System shutting down</td>
```

File: README.md [hunk 9]
```diff
@@ -1 +1,50 @@
...
+      <td>Blinks every second</td>
```
Comment (MasaakiWatanabeTier4) [inline README.md:29]:
```suggestion
      <td>Blink once every second</td>
```

File: README.md [hunk 10]
```diff
@@ -1 +1,50 @@
...
+    </tr>
+    <tr>
+      <td>Completes system shutdown</td>
```
Comment (MasaakiWatanabeTier4) [inline README.md]:
```suggestion
      <td>Power off</td>
```

File: README.md [hunk 11]
```diff
@@ -1 +1,50 @@
...
+      <td>Keeps going off</td>
```
Comment (MasaakiWatanabeTier4) [inline README.md]:
```suggestion
      <td>Off</td>
```

File: README.md [hunk 12]
```diff
@@ -1 +1,50 @@
...
+    </tr>
+  </tbody>
+</table>
+
+## Input and Output
+- input
+  - from [autoware_state_machine](https://github.com/eve-autonomy/autoware_state_machine)
```
Comment (MasaakiWatanabeTier4) [inline README.md]:
```suggestion
  - from [autoware_state_machine](https://github.com/eve-autonomy/autoware_state_machine/)
```

File: README.md [hunk 13]
```diff
@@ -1 +1,50 @@
...
+    - `/autoware_state_machine/state` : State of the system.
+- output
+  - to [dio_ros_driver](https://github.com/tier4/dio_ros_driver)
```
Comment (MasaakiWatanabeTier4) [inline README.md]:
```suggestion
  - to [dio_ros_driver](https://github.com/tier4/dio_ros_driver/)
```

File: README.md [hunk 14]
```diff
@@ -1 +1,50 @@
...
+    - `/dio/dout0` : GPIO output topic. (this topic is remapping from /ad_status_lamp_out)
```
Comment (MasaakiWatanabeTier4) [inline README.md]:
```suggestion
    - `/dio/dout0` : GPIO output topic. (this topic is remapped from `/ad_status_lamp_out`)
```

**Review by MasaakiWatanabeTier4 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md:4]: This node changes the lighting pattern of the lamp for the operation manager to check the system status.

**Pull request merged**